cpp/source/supercapacitor.templates.h

```diff
@@ -444,70 +444,4 @@ void SuperCapacitor<dim>::get_voltage(double &voltage) const
 {
   (*this->post_processor).get("voltage", voltage);
 }
-
-template <int dim>
-void SuperCapacitor<dim>::print_data(std::ostream &os) const
-{
-  double current;
-  double voltage;
-  (*this->post_processor).get("voltage", voltage);
-  (*this->post_processor).get("current", current);
-  os << boost::format("  %10.5f  %10.7f  \n") % current % voltage;
-
-  static int i                  = 0;
```

Comment:
What follows is how you trigger post processor output.
We need to save this somewhere. Maybe an inspector.